app/support/welcome-directs.ts

```diff
@@ -0,0 +1,189 @@
+import { readFile } from 'fs/promises';
+
+import createDebug from 'debug';
+import { loadAll } from 'js-yaml';
+import { z } from 'zod';
+import { DateTime, Duration } from 'luxon';
+import { fromError, isZodErrorLike } from 'zod-validation-error';
+
+import { dbAdapter, User, Job, Post, Comment, JobManager } from '../models';
+
+import { currentConfig } from './app-async-context';
+import { UUID } from './types';
+import { isNoEntryError } from './is-no-entry';
+
+export const WELCOME_DIRECT = 'WELCOME_DIRECT';
+
+export function initHandlers(jobManager: JobManager) {
+  jobManager.on(WELCOME_DIRECT, jobHandler);
+}
+
+const debugLog = createDebug('freefeed:welcomeDirects');
+const errorLog = createDebug('freefeed:welcomeDirects:errors');
```

Comment:
I like to use extend() to minimize the chance of typos:

```suggestion
const errorLog = debugLog.extend('errors');
```